website/content/en/docs/Community/Governance/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Governance
+weight: 10
+---
+
+<!-- The governance model is similar to https://github.com/containerd/project/blob/main/GOVERNANCE.md but simplified -->
+
+## Code of Conduct
+Lima follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+## Maintainership
+Lima is governed by Maintainers who are elected from active contributors.
+
+As a [Cloud Native Computing Foundation](https://cncf.io/) project, Lima will keep its [vendor-neutrality](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).
+
+### Roles
+Maintainers consist of two roles:
+
+- **Committer** (Full maintainership): Committers have full write accesses to repos under <https://github.com/lima-vm>.
+  Committers' commits should still be made via GitHub pull requests (except for urgent secuirity fixes), and they should not push commits directly.
+  Committers are also recognized as Maintainers in <https://github.com/cncf/foundation/blob/main/project-maintainers.csv>.
+
+- **Reviewer** (Limited maintainership): Reviewers may moderate GitHub issues and pull requests (such as adding labels and cleaning up spams),
+  but they do not have any access to merge pull requests nor push commits.
+  A Reviewer is considered to as a candidate of Committers.
+
+See also the [Contributing](../contributing) page.
+
+### Current maintainers
+
+| Name               | Role      | GitHub ID (not Twitter ID)                     | GPG fingerprint                                                                          |
+|--------------------|-----------|------------------------------------------------|------------------------------------------------------------------------------------------|
+| Akihiro Suda       | Committer | [@AkihiroSuda](https://github.com/AkihiroSuda) | [C020 EA87 6CE4 E06C 7AB9  5AEF 4952 4C6F 9F63 8F1A](https://github.com/AkihiroSuda.gpg) |
+| Jan Dubois         | Committer | [@jandubois](https://github.com/jandubois)     |                                                                                          |
+| Anders F Björklund | Committer | [@afbjorklund](https://github.com/afbjorklund) |                                                                                          |
+| Balaji Vijayakumar | Committer | [@balajiv113](https://github.com/balajiv113)   |                                                                                          |
```

Comment:
@lima-vm/maintainers Please add your GPG fingerprints, so that you all can be eligible to be a release manager (see the "Release process" section)

Reply:
Apparently it has been a while since I gave up on signing... `pub   rsa2048 2018-06-10 [SC] [expired: 2019-06-10]`

Reply:
Similar for me: `pub   rsa4096 2017-03-23 [SC] [expired: 2023-04-14]`.

I assume we can create a new key and upload it, even if it is not signed by other people?

Reply:
You can just renew expired keys https://filipe.kiss.ink/renew-expired-gpg-key/

Reply:
I have been using SSH signing key. I can create a new GPG key and upload it

Reply:
Will try to find and renew, otherwise will replace it.

Reply:
Thanks, added

Reply:
New key:
`5981 D2E8 4E4B 9197 95B3  2174 DC05 CAD2 E73B 0C92`

https://github.com/afbjorklund.gpg

Reply:
Thanks, added